src/index.test.js

```diff
@@ -67,6 +68,21 @@ describe('hotfix cmp.init', () => {
 		expect(window.guCmpHotFix.initialised).toBe(true);
 	});
 
+	it('warn if two versions are running simultaneously', () => {
+		const spy = jest.spyOn(global.console, 'warn');
```

Comment:
if this was mocked, i think it would stop it showing in the jest output while still making the same check?

Reply:
Yes, good idea!